README.md

```diff
@@ -0,0 +1,50 @@
+# Build Status Lights
+
+This is a Python server that uses Bitbucket Webhooks to display project build statuses on a [Nanoleaf](https://nanoleaf.me/) light panel.
+
+## Getting Started
+
+Right now, in order to use this server you need to be using Bitbucket build pipelines and have a Nanoleaf light panel (Aurora or Canvas) setup on your local network.
+
+Clone this repository
+
+```
+git clone https://github.com/astone123/build-status-lights.git
+```
+
+Configure the server by replacing example configuration values with your actual information
+
+```
+cp config.example.yaml config.yaml
+```
+
+Configuration values:
+
+- `projects`
+  - This is a list of the projects that you want to monitor with the lights. Each project needs to have a `repo_url` and `branch`.
```

Comment:
It can also include an array of `tile_ids` to specify the tiles you want to be associated with that projects.